src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -789,15 +791,15 @@ public void checkCourseAndExerciseGroupExclusivity(String entityName) throws Bad
      * <p>
      * Currently, exercise start dates are the same for all users
      *
-     * @return the time from which on access to the exercise is allowed, for exercises that are not part of an exam, this is just the release date.
+     * @return the time from which on access to the exercise is allowed, for exercises that are not part of an exam, this is just the release date or start date.
      */
     @JsonIgnore
-    public ZonedDateTime getIndividualReleaseDate() {
+    public ZonedDateTime getRepositoryUnlockDate() {
```

Comment:
Why is this code in the `Exercise` entity. Repositories just make sense for ProgrammingExercises?

Reply:
I've renamed it because it was only used at one place (and the new name fits better from my point of view, but feel free to check this). I didn't wanted to move the method in this PR.

Reply:
Reverted the renaming